terratorch/models/__init__.py

```diff
@@ -6,6 +6,8 @@
 from terratorch.models.smp_model_factory import SMPModelFactory
 from terratorch.models.timm_model_factory import TimmModelFactory
 
+from terratorch.models.smp_model_factory import get_smp_decoder
```

Comment:
I dont think we should expose this directly. Is there a good reason for importing this here?